drivers/base/bootvid/arm/bootvid.c

```diff
@@ -5,23 +5,23 @@
 
 PUSHORT VgaArmBase;
 PHYSICAL_ADDRESS VgaPhysical;
-BOOLEAN ClearRow = FALSE;
 
 /* PRIVATE FUNCTIONS *********************************************************/
 
 VOID
 NTAPI
-DisplayCharacter(IN CHAR Character,
-                 IN ULONG Left,
-                 IN ULONG Top,
-                 IN ULONG TextColor,
-                 IN ULONG BackColor)
+VidpDisplayCharacter(
```

Comment:
```suggestion
DisplayCharacter(
```
to emphasise it's a per-arch dependent global helper function and not a generic private bootvid helper.